modules/QuickEncounter.js

```diff
@@ -667,6 +667,9 @@ export class QuickEncounter {
         else {
             let openJournalSheet = null;
             for (let w of Object.values(ui.windows)) {
+				//Check to see if this is an Enhanced Journal window and get the subsheet
+				if(w.subsheet)
```

Comment:
Enhanced Journal is just a browser of Journal Entries, so to get access to the actual Journal Entry being displayed you'd need to check for a subsheet.